scripts/crash_test.py

```
@@ -0,0 +1,167 @@
+    for job_id in job_ids:
+        try:
+            response = requests.get(f"http://localhost:8001/api/jobs/{job_id}")
+            if response.status_code == 200:
+                status = response.json().get("status", "unknown")
+                if status == "completed":
+                    completed += 1
+                elif status == "failed":
+                    failed += 1
+                else:
+                    processing += 1
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)